andela/bp-andela-media-frontend
# #162376921 Design UI to view photos

#### What does this PR do?
Adds a component to view photos

#### Description of Task to be completed?
A user should be able to view photos

#### How should this be manually tested?
- Checkout to the branch
- Visit the route `photos`
- Confirm that the view is inline with this [mockup](https://projects.invisionapp.com/d/main/default/#/console/16015287/332188472/preview)

#### What are the relevant pivotal tracker stories?
[#162376921](https://www.pivotaltracker.com/story/show/162376921)

#### Any background context you want to add?
N/A

#### Screenshots
Coming soon
